web/src/components/index/UserTasks.module.css

```diff
@@ -0,0 +1,22 @@
+#chosenName {
```

Comment:
any reason this is has an id rather than being consistent?

Reply:
ah so no hypens! maybe just `.dialog` ?

Reply:
wilco
feels very wrong but I know the modules makes this possible

Reply:
hopefully we won't be having any other dialogs on this page

Reply:
I _think_ the name mangling means the only requirement is that there isn't another dialog in the component